crema/choices.py

```diff
@@ -32,6 +32,8 @@ class EventType(Enum):
 
     # AMS
     ASSESSMENT = "ASSESSMENT"
+    # topic for master user assessment data
+    USER_ASSESSMENT = "USER_ASSESSMENT"
```

Comment:
@rohitladdha good point

data to ASSESSMENT topic comes from AMS (legacy system)
data to USER_ASSESSMENT will come from Assessment Engine (new system developed by AMS team)

This is also a proposal from AMS team to have a separate topic.

There is overlap of certain data coming from AMS and Assessment Engine but:
- The names of the fields are different. For eg: There is a field called approver_type in data coming from AMA (legacy system). This field is now referred to as uptop_risk_segment in data coming from Assessment Engine (new system).
- AMS team does not send consistent data for fields that overlap between new and old systems.

Right now, their state is that AMS system sends ASSESSMENT data for first loans
and they are proposing that new system called Assessment engine will send data for refi.

I had a call with them and asked them to use the same topic for both first-loans and refi and have a way to distinguish between them using a existing field. They may have to make some changes on their end which is good long terms. If we use 2 different topics for first loans and refi it will be difficult for us to maintian.

Raqueeb is following this up with Akhil and will get back to us.

Reply:
Update: Refi data from Assessment engine will be sent to the new topic and later on the first loans data which currently goes to the ASSESSMENT topic will also be sent to this new topic.